runtime/autoload/man.vim

```diff
@@ -46,11 +46,13 @@ function! man#open_page(count, count1, mods, ...) abort
 
   call s:push_tag()
   let bufname = 'man://'.name.(empty(sect)?'':'('.sect.')')
+  set eventignore+=BufReadCmd
```

Comment:
Hmm. That's pretty nasty. Seems like we should recommend something else in the `:help :terminal` doc ...